src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -108,6 +108,8 @@ public class CreateGameController extends NodeController<Pane> {
 
   private final SimpleInvalidationListener createButtonStateListener = this::setCreateGameButtonState;
 
+  private MapVersionBean lastMapChosen = null;
+
```

Comment:
This field can be removed, also did you want to write a test for this?

Reply:
Oh yep Ill remove that and add some tests